classes/game.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'item'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(params)
+    super(params)
+
+    @multiplayer = params[:multiplayer]
+    @last_played_at = params[:last_played_at]
+  end
+
+  def can_be_archived?
+    @archived
+  end
```

Comment:
According to the project below requirements :

**- can_be_archived?() in the Game class
    - should override the method from the parent class
    - should return true if parent's method returns true AND if last_played_at is older than 2 years
    - otherwise, it should return false**
 
 it would be better to define the method as which should override its parent method because if its parents return true it will return the same result otherwise it will return false, you check the bellow code 👇

 def can_be_archived
    super && (Date.today - @played_at_date).to_i / 365 > 2
  end